src/table/table.scss

```
@@ -4,383 +4,373 @@
       }
-      font-style: italic;
-      cursor: not-allowed;
 
-      &:hover {
-        @include themed {
-          background-color: t(iui-color-background-1);
-        }
-
-        > .iui-tables-slot > .iui-tables-menu {
-          visibility: hidden;
-        }
+      &:hover > .iui-slot > .iui-more-options {
+        visibility: hidden;
       }
 
-      > .iui-tables-cell {
-        > img,
-        > svg,
-        > .iui-user-icon,
-        > .iui-user-icon > img {
-          filter: $iui-icons-color-multicolor-disabled;
-        }
+      img,
+      svg:not(.iui-radial),
+      .iui-user-icon {
+        filter: $iui-icons-color-multicolor-disabled;
       }
     }
+
+    &.iui-positive {
+      @include iui-table-row-status($status:positive);
+    }
+    
+    &.iui-warning {
+      @include iui-table-row-status($status:warning);
+    }
+
+    &.iui-negative {
+      @include iui-table-row-status($status:negative);
+    }
+    // #endregion Statuses & states
   }
 
-  .iui-tables-message-container {
+  // Empty & loading states
+  > .iui-table-empty {
+    text-align: center;
+    padding: $iui-xl;
     @include themed {
       color: t(iui-text-color-muted);
       background-color: t(iui-color-background-2);
     }
+  }
+}
+
+@mixin iui-table-cell {
+  align-items: center;
+  display: flex;
+  flex-grow: 1;
+  min-width: $iui-xxl;
+  min-height: $iui-baseline * 5;
+  padding-left: $iui-m;
+  overflow: hidden;
+  flex-basis: $iui-xxl;
+  position: relative;
+
+  &.iui-slot {
+    width: $iui-l * 2;
+    padding: 0;
+    flex-grow: 0;
+    min-width: 0;
     display: flex;
     justify-content: center;
-    height: 100%;
-    padding: $iui-xl;
+    align-items: center;
+    flex-basis: $iui-l * 2;
+  }
+
+  &.iui-positive {
+    @include iui-table-cell-status($status:positive);
+  }
+    
+  &.iui-warning {
+    @include iui-table-cell-status($status:warning);
   }
 
-  .iui-tables-paginator {
+  &.iui-negative {
+    @include iui-table-cell-status($status:negative);
+  }
+}
+
+@mixin iui-table-row-status($status) {
+  &,
+  + .iui-expanded-content {
     @include themed {
-      background-color: t(iui-color-background-3);
+      box-shadow: inset $iui-xxs 0 0 0 t(iui-icons-color-#{$status});
     }
-    user-select: none;
 
-    &:hover {
+    *::selection {
       @include themed {
-        background-color: t(iui-color-background-3) !important;
+        background-color: rgba(
+          t(iui-color-foreground-#{$status}-rgb),
+          t(iui-opacity-4)
+        );
       }
     }
+  }
 
-    > .iui-content {
-      @include themed {
-        border-bottom: solid 1px t(iui-color-background-4);
-      }
-      width: 100%;
-      display: flex;
-      justify-content: space-around;
-      align-items: center;
-      padding: $iui-table-density;
-      flex: 1;
-
-      > .iui-full-line {
-        flex: 1;
-      }
+  .iui-cell-end-icon > svg {
+    @include themed {
+      fill: t(iui-icons-color-#{$status});
```

Do we really want to do that? If yes, then it is not an end icon but a status icon.

---

Maybe we should only target the end icon within the main column?
```suggestion
      .iui-main-column .iui-cell-end-icon > svg {
          @include themed {
              fill: t(iui-icons-color-#{$status});
          }
      }
```